pkg/subctl/benchmark/latency.go

```diff
@@ -32,6 +32,12 @@ func StartLatencyTests(intraCluster bool) {
 	f = initFramework("latency")
 
 	if !intraCluster {
+		if framework.TestContext.GlobalnetEnabled {
```

Comment:
What would happen if user tries to run latency tests with intracluster on a Globalnet deployment?

Reply:
The tests should work properly in this case